jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-cluster-jbpm-schema.sql

```diff
@@ -0,0 +1,853 @@
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table CaseRoleAssignmentLog (
+        id bigint not null auto_increment,
+        caseId varchar(255),
+        entityId varchar(255),
+        processInstanceId bigint not null,
+        roleName varchar(255),
+        type integer not null,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table Content (
+        id bigint not null auto_increment,
+        content longblob,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table ContextMappingInfo (
+        mappingId bigint not null auto_increment,
+        CONTEXT_ID varchar(255) not null,
+        KSESSION_ID bigint not null,
+        OWNER_ID varchar(255),
+        OPTLOCK integer,
+        primary key (mappingId)
+    ) ENGINE=InnoDB;
+
+    create table TimerMappingInfo (
+    	id bigint generated not null auto_increment, 
+    	externalTimerId varchar(255), 
+    	kieSessionId bigint not null, 
+    	timerId bigint not null, 
+    	uuid varchar(255) not null, 
+    	primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table CorrelationKeyInfo (
+        keyId bigint not null auto_increment,
+        name varchar(255) not null,
+        processInstanceId bigint not null,
+        OPTLOCK integer,
+        primary key (keyId)
+    ) ENGINE=InnoDB;
+
+    create table CorrelationPropertyInfo (
+        propertyId bigint not null auto_increment,
+        name varchar(255),
+        value varchar(255),
+        OPTLOCK integer,
+        correlationKey_keyId bigint,
+        primary key (propertyId)
+    ) ENGINE=InnoDB;
+
+    create table Deadline (
+        id bigint not null auto_increment,
+        deadline_date datetime(6),
+        escalated smallint,
+        Deadlines_StartDeadLine_Id bigint,
+        Deadlines_EndDeadLine_Id bigint,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table Delegation_delegates (
+        pk_id int(10) auto_increment,
+        task_id bigint not null,
+        entity_id varchar(255) not null,
+        primary key (pk_id)
+    ) ENGINE=InnoDB;
+
+    create table DeploymentStore (
+        id bigint not null auto_increment,
+        attributes varchar(255),
+        DEPLOYMENT_ID varchar(255),
+        deploymentUnit longtext,
+        state integer,
+        updateDate datetime(6),
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table ErrorInfo (
+        id bigint not null auto_increment,
+        message varchar(255),
+        stacktrace varchar(5000),
+        timestamp datetime(6),
+        REQUEST_ID bigint not null,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table Escalation (
+        id bigint not null auto_increment,
+        name varchar(255),
+        Deadline_Escalation_Id bigint,
+        primary key (id)
+    ) ENGINE=InnoDB;
+
+    create table EventTypes (
+        pk_id int(10)auto_increment primary key,
+        InstanceId bigint not null,
+        element varchar(255)
+    ) ENGINE=InnoDB;
```

Comment:
Guess you forgot this. As `primary key` should be defined at the end of the create statement here like the others.

```
```
```suggestion
    create table EventTypes (
        pk_id int(10)auto_increment,
        InstanceId bigint not null,
        element varchar(255),
        primary key (pk_id)
    ) ENGINE=InnoDB;
```

Reply:
done